Amazon.QLDB.Driver/transaction/AsyncTransaction.cs

```diff
@@ -87,39 +90,9 @@ public async Task Commit()
                     throw new InvalidOperationException(ExceptionMessages.TransactionDigestMismatch);
```

Comment:
Random aside - what happens if this is ever thrown? Is the transaction committed on the server and we just don't know what happened? Aren't we sort of in unknown territory if this ever happens?

Reply:
Good point. Let's sync up with them on Tuesday.

It doesn't make much sense, now that I think about it.